src/app/layout.tsx

```diff
@@ -1,124 +1,217 @@
+                      />
+                    </svg>
+                    Home
+                  </a>
+                  <a
+                    href="#"
+                    className="mt-1 group flex items-center px-2 py-2 text-base leading-6 font-medium rounded-full hover:bg-[#343638] hover:text-gray-300"
+                  >
+                    <svg
+                      className="mr-4 h-6 w-6"
+                      fill="none"
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      stroke="currentColor"
+                      viewBox="0 0 24 24"
+                    >
+                      <path d="M15 17h5l-1.405-1.405A2.032 2.032 0 0118 14.158V11a6.002 6.002 0 00-4-5.659V5a2 2 0 10-4 0v.341C7.67 6.165 6 8.388 6 11v3.159c0 .538-.214 1.055-.595 1.436L4 17h5m6 0v1a3 3 0 11-6 0v-1m6 0H9"></path>
+                    </svg>
+                    Notifications
+                  </a>
+                  <a
+                    href="#"
+                    className="mt-1 group flex items-center px-2 py-2 text-base leading-6 font-medium rounded-full hover:bg-[#343638] hover:text-gray-300"
+                  >
+                    <svg
+                      className="mr-4 h-6 w-6"
+                      fill="none"
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      stroke="currentColor"
+                      viewBox="0 0 24 24"
+                    >
+                      <path d="M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z"></path>
+                    </svg>
+                    Profile
+                  </a>
+                  <a
+                    href="#"
+                    className="mt-1 group flex items-center px-2 py-2 text-base leading-6 font-medium rounded-full hover:bg-[#343638] hover:text-gray-300"
+                  >
+                    <svg
+                      className="mr-4 h-6 w-6"
+                      fill="none"
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      stroke="currentColor"
+                      viewBox="0 0 24 24"
+                    >
+                      <path d="M8 12h.01M12 12h.01M16 12h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"></path>
+                    </svg>
+                    More
+                  </a>
+                  <button className="bg-[#343638] w-48 mt-5 hover:bg-gray-600 text-white font-bold py-2 px-4 rounded-full">
                     Post
-                    </button>
+                  </button>
                 </nav>
                 <div className="flex-shrink-0 flex hover:bg-gray-00 rounded-full p-4 mt-12 mr-2">
-                    <Link href={profileURL} target="_blank" className="flex-shrink-0 group block">
```

Comment:
Note that I reverted this after your last PR. It's necessary to be here, because in yours there is an a > div > div > p > Link. So, conflicting link tags. It causes Next errors which are displayed on the left bottom side.

Reply:
I fixed